<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUp_dashboardAdminOverride</class>
 <widget class="QDialog" name="PopUp_dashboardAdminOverride">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PopUp_dashboardAdminOverride{
	background-color: #f2efff;
}</string>
  </property>
  <widget class="QFrame" name="basePopUpBodyFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#basePopUpBodyFrame {
	background-color: #e7e2ff;
	border: none;
	border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="adminoverride_labelDesc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#adminoverride_labelDesc {
	background-color: transparent;
	color: #4f4f4f;
}
</string>
    </property>
    <property name="text">
     <string>PLEASE ENTER A VALID PIN:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="adminoverride_fieldPin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#adminoverride_fieldPin{
	border: 1px solid #b3b3b3;  /* Black border */
    border-radius: 5px;        /* Rounded corners */
    padding: 5px;               /* Inner spacing */
    background-color: #f2efff;  /* Gray background */
    color: #000000; 
}

</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>XXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminoverride_buttonConfirmPin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#adminoverride_buttonConfirmPin {
    background-color: #141c52;
    color: white;
    border-radius: 20px;
    padding: 10px; /* Optional: Adjusts spacing inside the button */
}
#adminoverride_buttonConfirmPin:hover {
    background-color: #1a237e; /* Slightly lighter blue when hovered */
}
#adminoverride_buttonConfirmPin:pressed {
    background-color: #0f1442; /* Darker blue when clicked */
}</string>
    </property>
    <property name="text">
     <string>CONFIRM PIN</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="basePopUpTitleFrame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-10</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#basePopUpTitleFrame {
	background-color: #141c52;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_titleAdminOverride">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_tableTitleDashboard {
	background-color: transparent;
	color: #FFFFFF;
}
</string>
    </property>
    <property name="text">
     <string>ADMIN OVERRIDE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return_to_youraccount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_return_to_youraccount {
    background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
